metaflow/plugins/kfp/tests/run_integration_tests.py

```diff
@@ -218,9 +224,10 @@ def exponential_backoff_from_kfam_errors(kfp_run_cmd: str, correct_return_code:
             shell=True,
         )
 
-        if "Reason: Unauthorized" in run_and_wait_process.stderr or "Failed to connect to the KFAM service" in run_and_wait_process.stderr:
-            print(f"KFAM issue encountered. Backing off for {interval} seconds...")
-            continue
+        for platform_error_message in platform_error_messages:
+            if platform_error_message in run_and_wait_process.stderr:
+                print(f"KFAM issue encountered. Backing off for {interval} seconds...")
```

Comment:
Can you just print out the `run_and_wait_process.stderr` instead of hard coding that it was the `KFAM issue`?